lib/style-info.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path')
+const { workspace } = require('vscode')
+const fs = require('fs')
+
+let usesStandard = true
+let eslintCLI = null
+
+const styleInfoPromise = new Promise((resolve, reject) => {
```

Comment:
use mz/fs here instead of fs. That way you don't have to construct a new Promise